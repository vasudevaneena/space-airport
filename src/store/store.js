import {createStore, applyMiddleware} from 'redux';
import thunk from 'redux-thunk';

// import rootReducer from '../redux/rootReducer';
import reducer from '../redux/spaceReducer';
const store = createStore(reducer,applyMiddleware(thunk))
export default store;