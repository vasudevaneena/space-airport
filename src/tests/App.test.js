import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, mount } from "enzyme";
import { Provider } from "react-redux";
import store from "../store/store";
import Container from "../components/display/DisplayConsole";

configure({ adapter: new Adapter() });

let wrapper;

beforeEach(() => {
  wrapper = mount(
    <Provider store={store}>
      <Container />
    </Provider>
  );
});

describe("Container snapshot", () => {
  test("matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
