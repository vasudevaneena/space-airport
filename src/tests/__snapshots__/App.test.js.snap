// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container snapshot matches snapshot 1`] = `ReactWrapper {}`;
